Can't get `eventLoopBlockIntegration` from  `@sentry/electron/native`

### Is there an existing issue for this?

- [x] I have checked for existing issues https://github.com/getsentry/sentry-javascript/issues
- [x] I have reviewed the documentation https://docs.sentry.io/
- [x] I am using the latest SDK release https://github.com/getsentry/sentry-javascript/releases

### How do you use Sentry?

Sentry Saas (sentry.io)

### Electron SDK Version

6.10.0

### Electron Version

33.3.1

### What platform are you using?

Windows

### Link to Sentry event

_No response_

### Steps to Reproduce

1. I added a new test as an example: "import { eventLoopBlockIntegration } from '@sentry/electron/native';"
2. Updated TypeScript version to 5.4.5.
3. Changed the moduleResolution in tsconfig.json to Node16.

<img width="655" height="246" alt="Image" src="https://github.com/user-attachments/assets/ff8c28fe-0da6-4193-8425-8b2708fcd7e8" />

<img width="598" height="296" alt="Image" src="https://github.com/user-attachments/assets/6fe3b133-913a-469c-8c65-1e253db39d40" />

<img width="771" height="127" alt="Image" src="https://github.com/user-attachments/assets/3acc68e5-19f2-410b-91e6-09b93850413f" />

### Expected Result

I want to get eventLoopBlockIntegration successful 

### Actual Result

But build error:
$ tsc && webpack --config webpack.config.js
src/sentryReportMain.ts:2:43 - error TS2307: Cannot find module '@sentry/electron/native' or 
its corresponding type declarations.

2 import { eventLoopBlockIntegration } from '@sentry/electron/native';
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/sentryReportMain.ts:2

error Command failed with exit code 2.
----------------------------------------------------------------------------------------------------

Another question concerns the suggested import statement. 
The documentation at https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/event-loop-block/ recommends using "import { eventLoopBlockIntegration } from "@sentry/node-native";". 
However, the GitHub pull request at https://github.com/getsentry/sentry-electron/pull/1188 suggests using "import { eventLoopBlockIntegration } from '@sentry/electron/native';". 
This discrepancy raises questions about which import statement is appropriate for different contexts.

--- a/.npmignore
+++ b/.npmignore
@@ -4,8 +4,8 @@
 !/esm/**/*
 !/preload/**/*
 !/main/**/*
+!/native/**/*
 !/renderer/**/*
 !/common/**/*
 !/utility/**/*
 !/index.*
-!/integrations.*

